src/python/py/models/README.md

```diff
@@ -185,6 +185,18 @@ python3 -m onnxruntime_genai.models.builder -i path_to_local_folder_on_disk -o p
 python3 builder.py -i path_to_local_folder_on_disk -o path_to_output_folder -p precision -e execution_provider -c cache_dir_to_store_temp_files --extra_options use_8bits_moe=1
 ```
 
+#### Use QDQ Pattern for Quantization
```

Comment:
Can you add a reference to this section in the table of contents after the QMoE reference?

https://github.com/microsoft/onnxruntime-genai/blob/0f193ac0561726d8432cffcda1be55652a390814/src/python/py/models/README.md?plain=1#L15-L20